docs/man_pages/project/testing/debug-ios.md

```diff
@@ -2,8 +2,18 @@
 title: tns debug ios
 position: 5
 ---<% } %>
+
 # tns debug ios
 
+### Description
+
+Initiates a debugging session for your project on a connected iOS device or in the iOS simulator. When necessary, the command will prepare, build, deploy and launch the app before starting the debug session. While debugging, the output from the application is printed in the console and any changes made to your code are synchronizes with the deployed app. <% if(isHtml) { %>Any debugging traffic is forwarded to port 8080 (or the next available one) from the device to the local machine.<% } %>
+
+<% if(isConsole && (isWindows || isLinux)) { %>WARNING: You can run this command only on macOS systems. To view the complete help for this command, run `$ tns help debug ios`<% } %>
+<% if((isConsole && isMacOS) || isHtml) { %>
+<% if(isHtml) { %>> <% } %>IMPORTANT: Before building for iOS device, verify that you have configured a valid pair of certificate and provisioning profile on your macOS system. <% if(isHtml) { %>For more information, see the [Code Signing](https://developer.apple.com/support/code-signing/) and [Maintain Signing Assets](https://help.apple.com/xcode/mac/current/#/dev3a05256b8) sections from the Apple Developer documentation.<% } %>
+
+### Commands
```

Comment:
```
Deploy on device/simulator, run the app, start Safari Web Inspector and attache the debugger.
``` 
is incorrect. Should be something like that
```
Deploy on device/simulator, run the app and attach the debugger to Chrome Developer Tools.
```

Reply:
Does `Deploy on device/emulator, run the app and generate a Chrome DevTools link for debugging` sounds okay to you? Or perhaps `Deploy on device/emulator, run the app, attach a debugger and generate a Chrome DevTools link for debugging` sounds more correct (but longer)?

Reply:
I'm voting for the second option 
```
Deploy on device/emulator, run the app, attach a debugger and generate a Chrome DevTools link for debugging.
```